### PR TITLE
Reviews model controller

### PR DESCRIPTION
**Pull request by Jide-Muritala**
- Create new review is now working
- I will include reviews on flat show page after Fran merges the flat show page he is working on with master
- I also made a few changes to application.html related to how "Hi current.user" is displayed. I changed h4 to p (h4 looked too big), added an exclamation mark and a demarcation line and put it in a div container to move it away from the margin.

Capture

### CODE REVIEW

**Comment by margeol90** [comment]
This all looks good to me. Great work @Jide-Muritala !
Can we discuss later the relationship between a review and a booking? Based on our schema, we had linked Reviews to Bookings, so a Flat has many reviews through bookings?

**Comment by Jide-Muritala** [comment]
Yes, I forgot to include through bookings. So, I will probable close this and start afresh after we have the booking model rather than trying to correct this one in a complicated way.

File: app/controllers/reviews_controller.rb [hunk 1]
```diff
@@ -0,0 +1,28 @@
+class ReviewsController < ApplicationController
+  before_action :set_flat, only: %i[new create]
+
+  def new
+    @flat = Flat.find(params[:flat_id])
```
Comment (margeol90) [inline app/controllers/reviews_controller.rb:5]: if you do set_flat do you need this line?

Comment (Jide-Muritala) [reply app/controllers/reviews_controller.rb:5]: It is no longer needed

**Pull request merged**